src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -46,18 +46,21 @@ const MapView: React.FC<MapViewProps> = ({ blocks, neighborhoods, sites }) => {
   // block id for modal
   const [activeBlockId, setActiveBlockId] = useState<number>(-1);
   // logic for reservation modal to complete action selected by user
-  const handleOk = (): void => {
+  const handleOk = (team?: number) => {
```

Comment:
Would be nice to move `handleOk` into the reservation modal file and have `showModal` be a state of the reservation modal that updates to true whenever a new block ID is passed using `useEffect` - there's a lot of code in `MapView` that we want to cut down and move elsewhere